components/charts/doughnut.js

```diff
@@ -35,9 +35,9 @@ export default function DoughnutChart(props) {
 
   return (
     <div>
-      <div className="header">
+      {/* <div className="header">
```

Comment:
Are these comments going to be needed in the future? If not they should be removed

Reply:
they might be, they are headings but I was unsure if we should keep them or not as we have actual segment headings instead.